services/dqa-secondary-reports/main.py

```diff
@@ -158,6 +176,14 @@ def versioned_data():
 
         return versioned_data
 
+    def serve_expanded_site_totals(self, version, site):
+        "Entrypoint for Flask routing for site totals for a given version of PEDSnet data model"
```

Comment:
This has the docstring as the method below. What is different about this *expanded* version?